Homebrew/Homebrew.munki.recipe.yaml

```diff
@@ -28,28 +28,40 @@ Process:
         installcheck_script: |
           #!/bin/zsh
 
+          autoload is-at-least
```

Comment:
Neat ✨

Reply:
Got this from https://scriptingosx.com/2019/11/comparing-version-strings-in-zsh/